apps/chat/src/components/Common/ApplicationWizard/CodeAppView/CodeEditor.tsx

```diff
@@ -129,6 +129,7 @@ const CodeEditorView = ({
         scrollbar: {
           alwaysConsumeMouseWheel: false,
         },
+        automaticLayout: true,
```

Comment:
it's true by default as I understand. Am I right?

Reply:
it's said that's no
https://stackoverflow.com/questions/47017753/monaco-editor-dynamically-resizable
![image](https://github.com/user-attachments/assets/534fcac7-a93e-4dc3-b82f-ac29afc064fe)